docs-requirements.txt

```diff
@@ -0,0 +1,8 @@
+sphinx~=2.4
+sphinx-rtd-theme~=0.4
+sphinx-autodoc-typehints~=1.10.2
+# External
+opentelemetry-api
+opentelemetry-sdk
+psutil
```

Comment:
I get errors if I don't include `psutil` but I can't seem to find where in the docs it is needed.

Reply:
psutil is used in PerformanceMetrics class, docs need all external libraries used in the code to be able to build